Sources/Orders/Orders.docc/GettingStarted.md

```diff
@@ -0,0 +1,304 @@
+# Getting Started with Orders
+
+Create the order data model, build an order and distribute it.
+
+## Overview
+
+The Orders framework provides models to save all the basic information for orders, user devices and their registration to each order.
+For all the other custom data needed to generate the order, such as the barcodes, merchant info, etc., you have to create your own model and its model middleware to handle the creation and update of order.
+The order data model will be used to generate the `order.json` file contents.
+
+The order you distribute to a user is a signed bundle that contains the `order.json` file, images, and optional localizations.
+The Orders framework provides the ``OrdersService`` class that handles the creation of the order JSON file and the signing of the order bundle, using an ``OrdersDelegate`` that you must implement.
+The ``OrdersService`` class also provides methods to send push notifications to all devices registered when you update an order, and all the routes that Apple Wallet uses to retrieve orders.
+
+### Implement the Order Data Model
+
+Your data model should contain all the fields that you store for your order, as well as a foreign key to ``Order``, the order model offered by the Orders framework.
+
+```swift
+import Fluent
+import struct Foundation.UUID
+import Orders
+
+final class OrderData: OrderDataModel, @unchecked Sendable {
+    static let schema = "order_data"
+
+    @ID
+    var id: UUID?
+
+    @Parent(key: "order_id")
+    var order: Order
+
+    // Example of other extra fields:
+    @Field(key: "merchant_name")
+    var merchantName: String
+
+    // Add any other field relative to your app, such as an identifier, the order status, etc.
+
+    init() { }
+}
+
+struct CreateOrderData: AsyncMigration {
+    public func prepare(on database: Database) async throws {
+        try await database.schema(Self.schema)
```

Comment:
I don't think using `Self.schema` compiles as it's likely referencing the migration, let's use `OrderData.schema`